.github/renovate.json5

```diff
@@ -1,6 +1,6 @@
 {
   "extends": [
-    "github>newrelic/coreint-automation:renovate-base.json5"
+    "github>newrelic/k8s-metadata-injection:renovate-base.json5"
   ],
   ignoreDeps: ["ingress-nginx/kube-webhook-certgen"],
```

Comment:
If you are including this line on the shared config, it is not needed here anymore

Reply:
Ah that's right. I'll remove it.